path/_interface.ts

```diff
@@ -3,14 +3,22 @@
 
 /**
  * A parsed path object generated by path.parse() or consumed by path.format().
+ * parse("c:\\path\\dir\\index.html")
```

Comment:
Can you use code block and `@example` directive here?

Reply:
Thanks for the feedback. Is there any guideline or convention about this? The contributing guideline doesn't talk about this

https://github.com/denoland/deno_std/blob/main/.github/CONTRIBUTING.md